spec/index.html

```diff
@@ -283,6 +283,14 @@ <h2>Introduction</h2>
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
+    <p>
+      This document is based on YAML 1.2.2,
+      but YAML-LD is not tied to a specific version of YAML.
+      Implementers concerned about features related to a specific YAML version
+      can specify it in documents using the `%YAML` directive
+      (see <a href="#int" class="sectionRef"></a>).
+    </p>
+
```

Comment:
From https://github.com/json-ld/yaml-ld/commit/2c26c46f3ab4e07f01988164d3d166119456f855#r77665535, add something about transitive reflection of errors and warnings when parsing YAML.